External/Plugins/ProjectManager/Projects/Project.cs

```diff
@@ -76,6 +75,7 @@ public Project(string path, CompilerOptions compilerOptions)
         }
 
         public abstract string Language { get; }
+        public abstract string LanguageDisplayString { get; }
```

Comment:
I don't think anybody to have created a plugin adding a new project type, so that should be ok.